### PR TITLE
Whisper podcast transcripts for Chaincode Podcast

### PR DESCRIPTION
**Pull request by Beige-Coffee**
Submitting a pull request to add the first iteration of AI-generated transcripts. Please note the following:

- Transcripts were generated using a combination of Whisper (https://github.com/openai/whisper) and PyAnnote (https://github.com/pyannote/pyannote-audio)
- Speaker identification does not include the name of the speaker. It only includes "SPEAKER_00", "SPEAKER_01", etc.
- Some Bitcoin-specific words are transcribed correctly, while others are not. The next iteration of the model will seek to perform better on Bitcoin words.

Please let me know if there are any other comments or suggestions to be incorporated into the next iteration of the model & transcriptions.

### CODE REVIEW

File: chaincode-labs/chaincode-podcast/2020-03-12-Matt-Corallo-and-Compact-Blocks-FIBRE-Episode-6.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+SPEAKER_01: So today we actually had a slew of things we were going to talk to him about we gave him eight different options to look Over and he chose four and then we only talked about two So we're going to talk a little bit about compact blocks and the fiber network that he set up
+
+SPEAKER_03: Very cool.
+
+SPEAKER_01: Enjoy. Welcome, Matt. Hi. Hi, Matt. So today we're gonna do a little bit of this is your life, Bitcoin. And- I'm not dead yet. And so we've dug up some, I mean, you have a lot of contributions over the years, and so there's lots to talk about, but we'll start with three. Maybe let's start with compact blocks. So tell us a little bit about what compact blocks are, and then we can dive a little bit deeper.
+
+SPEAKER_02: you
+
+SPEAKER_03: Bye.
```
Comment (adamjonas) [inline chaincode-labs/chaincode-podcast/2020-03-12-Matt-Corallo-and-Compact-Blocks-FIBRE-Episode-6.md:30]: you. bye.

File: chaincode-labs/chaincode-podcast/2020-11-09-Murch-and-Enterprise-Wallets-UTXO-Management-Episode-8.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+transcript_by: Whisper AI & PyAnnote
+categories: ['podcast']
+tag: ['Enterprise UTXO management', 'The impact of the 2017 hype cycle', 'The importance of UTXO set minimization', 'The fee market today', 'Batching TransactionsPayment Batching', 'ConsolidationsConsolidation of 4 Million UTXOs at Xapo', 'Change splitting', 'Replace By FeeRBF in the wildFee bumping', 'SegWit', 'How enterprises estimate fees', 'Omnibus wallets', 'Off-chain sending', 'Taproot', 'Bitcoin Optech Field Report: How segwit and batching could have saved half a billion dollars in fees']
+---
+
+Chaincode Labs podcast: Murch and Enterprise Wallets/UTXO Management - Episode 8
+
+SPEAKER_02: to throw out a few numbers there, non-segwit inputs cost almost 300 bytes, and native segwit inputs cost slightly more than 100 bytes. There's almost a reduction by two-thirds in fees if you switch from non-segwit to native segwit.
+
+SPEAKER_00: Hi everyone, welcome to the ChainCode podcast. My name is Kara Lee. And it's Jonas. And we're back. We're back. We're not all back. So, while Jonas and I are coming to you very socially distanced from the New York office,
```
Comment (adamjonas) [inline chaincode-labs/chaincode-podcast/2020-11-09-Murch-and-Enterprise-Wallets-UTXO-Management-Episode-8.md:12]: whisper is really struggling with Caralie.

File: chaincode-labs/chaincode-podcast/2020-11-30-Carl-Dong-and-Build-Systems-Episode-9.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
+---
+title: Carl Dong and Build Systems - Episode 9
+transcript_by: Whisper AI & PyAnnote
+categories: ['podcast']
+tag: ['GUIX talk at 2019 Breaking Bitcoin about reproducible builds', 'Gitian builds', 'Fake time', 'Timestamps and reproducibility of packages', 'reproducible-builds.org SOURCE_DATE_EPOCH', 'Toolchains', 'Windows Builds and reproducibility GCC libtool pointer confusion', 'NSIS', 'Using Debian as an example', 'User choice in their security model', 'Making the process accessible', 'Mac OS X Toolchain & SDK and cross-compiling']
+---
+
+Chaincode Labs podcast: Carl Dong and Build Systems - Episode 9
```
Comment (adamjonas) [inline chaincode-labs/chaincode-podcast/2020-11-30-Carl-Dong-and-Build-Systems-Episode-9.md:8]: for comparison: https://btctranscripts.com/chaincode-labs/chaincode-podcast/2020-11-30-carl-dong-reproducible-builds/

File: chaincode-labs/chaincode-podcast/2021-05-13-Matt-Corallo-and-Lightning-Development-Kit-Episode-13.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
+---
+title: Matt Corallo and Lightning Development Kit - Episode 13
+transcript_by: Whisper AI & PyAnnote
+categories: ['podcast']
+tag: ['Starting Rust-Lightning', 'Language bindings challenges FFI', 'Interoperability of Lightning', 'Zero-value invoices', 'Keysend/push payments/spontaneous payments', 'What is the LDK stack?', 'Anchor Outputs Child pays for parent (CPFP)', 'Who tracks the onchain state when using LND?', 'LDK tracks the channel commitment transaction, how is that done? Compact block filters', 'Contrasting multiple implementations working from a spec in Lighting vs. no spec with one dominant reference implementation in Bitcoin The Lightning Spec', "What's the state of the Lightning Network? Have we moved beyond #reckless? Denial of Service (DoS) vulnerabilities Channel Jamming", 'Eltoo and the punishment dynamic', 'Will the network trend to trusted relationships and lend itself to KYC?', 'FATF updates its guidance on Virtual Asset Service Provider (VASP)', 'Where LDK goes from here?']
+---
+
+Chaincode Labs podcast: Matt Corallo and Lightning Development Kit - Episode 13
```
Comment (adamjonas) [inline chaincode-labs/chaincode-podcast/2021-05-13-Matt-Corallo-and-Lightning-Development-Kit-Episode-13.md:8]: for comparison: https://btctranscripts.com/chaincode-labs/chaincode-podcast/2021-05-12-matt-corallo-ldk/

File: chaincode-labs/chaincode-podcast/2020-03-12-Matt-Corallo-and-Compact-Blocks-FIBRE-Episode-6.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
+---
+title: Matt Corallo and Compact Blocks/FIBRE - Episode 6
+transcript_by: Whisper AI & PyAnnote
+categories: ['podcast']
+tag: []
+---
+
+Chaincode Labs podcast: Matt Corallo and Compact Blocks/FIBRE - Episode 6
```
Comment (adamjonas) [inline chaincode-labs/chaincode-podcast/2020-03-12-Matt-Corallo-and-Compact-Blocks-FIBRE-Episode-6.md:8]: for comparison: https://btctranscripts.com/chaincode-labs/chaincode-podcast/2020-03-12-matt-corallo-compact-blocks-fibre/

**Review by adamjonas — COMMENTED** [review]
This is going to take some human intervention but it's a great starting point.

Some questions:
1. we need humans for the diarization - how do we plan to do that?
2. > Some Bitcoin-specific words are transcribed correctly, while others are not. The next iteration of the model will seek to perform better on Bitcoin words.

What is your approach here? Do you plan to train the models with bitcoin terms or do some more basic post-processing find and replace?
3. For tags, I might suggest running the text through some classifiers with a list of the [current transcript tags](https://btctranscripts.com/tags/) as suggestions. Better yet, I really like the tags from [bitcoin StackExchange](https://bitcoin.stackexchange.com/tags). Without constraints, some of those tags get pretty offbase.
4. You submitted a few transcripts that are already transcribed and reviewed. If we are going to do this in large scale, how are we going to ensure we aren't double submitting?

**Review by kouloumos — COMMENTED** [review]
> This is going to take some human intervention but it's a great starting point.

🚀

> Some questions:
> 
> 1. we need humans for the diarization - how do we plan to do that?

I was thinking of a separate category (maybe it could be accomplished with the `draft: True` metadata field) alongside a Call-to-Action at the homepage. A banner could also go on top of the transcript's page when `draft`, again with a CTA. From there, we could have a GitHub umbrella issue for people to show interest (to avoid duplicate work).

If we are planning for A LOT of whisper transcripts, we could tweak a bit the https://github.com/bitcointranscripts/bitcointranscripts/issues/171 idea, and come up with a contributors board (automated process that updates a json file when a PR is merged). End of each month, first N contributors get a few sats. That said, we could first start with the original idea to see if interest actually exist. 

Last, but imo more important, if we need more humans, it means that we need to maximize awareness. Maybe this is a nice opportunity to discuss the switch to the [@btctranscripts](https://twitter.com/btctranscripts) twitter account. And from there we could discuss some tweet bot strategies to direct interest to the work that needs to be done for the whisper transcripts.

**Comment by adamjonas** [comment]
closing this experiment now that we are up and running

**Pull request closed without merging**